jac-uk/digital-platform
# #1310 data archiving soft deletes

## What's included?
Soft deleting feature so deleted records are archived in `{{collection_name}}_deleted` collection within firestore

## Who should test?
✅ Developers

## How to test?

Create / find a record that you can delete then delete it. 

Then check the `{{collection_name}}_deleted` collection and check if the deleted record exists.

## Risk - how likely is this to impact other areas?
Low risk

## Additional context


---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
